src/util-lua-common.c

```diff
@@ -666,6 +666,28 @@ static int LuaCallbackFileState(lua_State *luastate)
 }
 
 /** \internal
+ *  \brief wrapper to force magic lookup on all files from lua script
+    \retval cnt number of items placed on the stack
+ */
+static int LuaCallbackFileForceMagic(lua_State *luastate)
+{
+    FileForceMagicEnable();
```

Comment:
This is really not meant to be run at packet runtime, as it modifies a global variable that doesn't use locking or atomics. It *should* be fairly harmless though.